torch/testing/_internal/common_quantization.py

```diff
@@ -1648,6 +1649,12 @@ def forward(self, x):
         x = self.fc2(x)
         return self.dequant(x)
 
+class ManualLinearDynamicQATModel(ManualLinearQATModel):
```

Comment:
the forward for dynamic needs to be different from normal qat linear since dynamic doesn't need quant/dequant.